devel/fortio/grpcrunner_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Adapted from istio/proxy/test/backend/echo with error handling and
+// concurrency fixes and making it as low overhead as possible
+// (no std output by default)
+
+package fortio
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/health"
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+// DynamicGrpcHealthServer starts and returns the port where a Grpc Health
+// server is running.
+func DynamicGrpcHealthServer() int {
```

Comment:
nit: i might call this `Start...` just to clarify that it schedules a local server.